Lib/test/test_capi/test_long.py

```diff
@@ -588,6 +588,21 @@ def test_long_fromnativebytes(self):
                     self.assertEqual(expect_u, fromnativebytes(v_be, n, -1, 0),
                         f"PyLong_FromUnsignedNativeBytes(buffer, {n}, <native>)")
 
+    def test_long_sign(self):
+        # Test PyLong_Sign()
+        check = _testcapi.pylong_sign
```

Comment:
Can you please rename the variable to pylong_sign or sign? "check" sounds weird here.

Reply:
Indeed.  Done.